docs/Anjay_integrations/Anjay_ST_integration/B-U585I-IOT02A-BG96.md

```diff
@@ -0,0 +1,90 @@
+    - In **Select root directory**, indicate the catalog containing the cloned Anjay freeRTOS client repository.
+    - In the **Projects** field, select **Anjay-freertos-client-B-U585I-IOT02A-BG96** and click **Finish**.
+    ![Import project](images/import.PNG "Import project")
+0. In the Project Explorer, navigate to the **Anjay-freertos-client-B-U585I-IOT02A-BG96** project:
+    - Right-click on the project name and select **Build Project**. The build should take less than one minute to complete.
+    - After the build is finished, right-click on the project name, select **Run As** and click the **1 STM32 Cortex-M C/C++ Application** option.
+        - In the **Lauch Configuration Selection**, choose the **Anjay-freertos-client-B-U585I-IOT02A-BG96** option and click **OK**.
+0. After the build and run are complete, the board is now compiled.
+
+## Connecting to the LwM2M Server
+
+To connect to Coiote IoT Device Management LwM2M Server, please register at https://eu.iot.avsystem.cloud/.
+
+!!! note
+    If you use BG96-based configuration, you must upgrade the firmware of the modem to at least the `BG96MAR02A08M1G` revision. Older versions may cause unexpected loss of connection.
+
+To connect the board:
+
+1. Log in to Coiote DM and from the left side menu, select **Device Inventory**.
+2. In **Device Inventory**, click **Add device**.
+3. Select the **Connect your LwM2M device directly via the Management server** tile.
+       ![Add via Mgmt](images/mgmt_tile.png "Add via Mgmt")
+    3. In the **Device credentials** step:
+         - In the **Device ID** enter your board endpoint name, e.g. `test_device`.
+             ![Device credentials step](images/add_mgmt_quick.png "Device credentials step")
+         - In the **Security mode** section, select the **PSK (Pre-Shared Key)** mode:
+              - In the **Key identity** field, type the same name as in the `Endpoint name` field
+              - In the **Key** field, type the shared secret used in the device-server authentication.
+    4. Click the **Add device** button and **Confirm** in the confirmation pop-up.
+    5. In the **Connect your device** step, wait for the board to connect.
+    6. Click **Go to device** to see your added device dashboard.
+
+    !!! tip
+        LwM2M Server URI, endpoint name and other informations you can be found in the **configuration** tab.
+
+## Configuring the Client
+
+1. With the board still connected to a serial port interface, open a serial communication program.
+2. Press the reset button located on the board. This should trigger the following prompt:
+
+    ``Press any key in 3 seconds to enter config menu...``
+
+3. Press any key and in the configuration menu, change the default credentials to your data by following the instructions presented in the program and save it.
+   ![Client configuration](images/config_menu1.png "Client configuration"){:style="float: left;margin-right: 1177px;margin-top: 17px;margin-bottom: 17px;"}
```

Comment:
I would change/update the screenshot, in the screenshot e.g. endpoint_name is `michal-freertos`, but earlier in the tutorial in the Coiote DM we have `test_device`, that can confuse some people.